projects/spectator/src/lib/matchers.ts

```diff
@@ -52,7 +52,7 @@ const hasCss = (el: HTMLElement, css: { [key: string]: string }) => {
         continue;
       }
 
-      if (trim($el.css(prop)) !== trim(value) && trim(el.style[prop]) !== trim(value)) {
+      if (trim($el.css(prop) || '') !== trim(value) && trim(el.style[prop] || '') !== trim(value)) {
```

Comment:
It makes sense to add this protection in the trim function.

Reply:
Are we sure about all use cases? 
Anyway all tests are passed with this, so gonna push now, thanks!